internal/loadgen/eventhandler/transport.go

```diff
@@ -83,6 +74,20 @@ func (t *Transport) sendEvents(req *http.Request, r io.Reader, ignoreErrs bool)
 	return nil
 }
 
+func logResponseOutcome(logger *zap.Logger, res *http.Response) {
```

Comment:
question: should this function guard for the `res=nil` case? In the way it is used in `SendV2Events` this case should never happen, but in case of other uses could lead to a runtime panic.

Reply:
Ah good point, because since a standalone function can be called anywhere, not only after the error check as in `sendEvents`.
will amend

Reply:
added